### ISSUE TITLE
Retrieving category translation generating error

### ISSUE THREAD

**Issue by daniyals14** [valu-digital/wp-graphql-polylang#15]
Hello, im trying to retrieve translations on category and it's giving me the following internal error, there are no PHP error log for this error to debug further. There exists a valid translated category im passing the ID for.

Graphql:
`query MyQuery {
  category(idType: DATABASE_ID, id: 2) {
    name
    translations {
      id
    }
  }
}`

Error:`{
  "errors": [
    {
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 5,
          "column": 7
        }
      ],
      "path": [
        "category",
        "translations",
        0,
        "id"
      ]
    }
  ],
  "data": {
    "category": {
      "name": "xyz",
      "translations": [
        null
      ]
    }
  }
}`

**Comment by esamattis** [comment]
I'm not able reproduce this. 

Can you add this to wp-config.php

```php
define('GRAPHQL_DEBUG', true);
```

and run it again?

Or better yet do have some steps to what kind of data should be in the database?

**Comment by daniyals14** [comment]
Added the graphql_debug in the wp-config and got the following message

```
"errors": [
    {
      "debugMessage": "Cannot return null for non-nullable field Category.id.",
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 5,
          "column": 7
        }
      ],
      "path": [
        "category",
        "translations",
        0,
        "id"
      ]
    }
  ],
```

In terms of data, i have a category with a translated category in arabic language, both the categories have posts in them.

**Comment by esamattis** [comment]
I think I figured it out. Can you try the version from the master branch?

**Comment by daniyals14** [comment]
works perfect on master! thanks for the quick fix!

**Comment by esamattis** [comment]
Thanks for reporting!

I released the fix in v0.3.8 to Packagist

https://github.com/valu-digital/wp-graphql-polylang/releases/tag/v0.3.8
